feat: :sparkles: Wallet functionality for ICRC 1 tokens

Jair-MoralesN opened the pull request:
Wallet to manage their digital assets and perform transactions with major ICRC1 cryptocurrencies

User-friendly interface for ICRC 1 tokens

timohanke commented on README.md:
````diff
@@ -1 +1,107 @@
+1. Clone the repository to your local machine:
+
+   ```
+   git clone https://github.com/SDS-Timo/wallet.git
````
git clone git@github.com:research-ag/wallet.git

timohanke commented on README.md:
````diff
@@ -1 +1,107 @@
+   npm install
+   ```
+
+4. Create .env file in the root of the proyect:
````
proyect -> project

timohanke commented on README.md:
```diff
@@ -1 +1,107 @@
+   NODE_ENV=PRODUCTION
+   # Ledger canister for ICP
+   VITE_ICP_LEDGER_CANISTER_ID="ryjl3-tyaaa-aaaaa-aaaba-cai"
+   # Transacction history API for ICP
```
transacction -> transaction

timohanke commented on frontend/assets/svg/files/decred-icon.svg:
delete the file

timohanke commented on frontend/assets/img/fnid-logo.png:
should this file be named nfid-logo.png?

timohanke commented on frontend/assets/svg/files/binance-icon.svg:
delete this file

timohanke commented on frontend/assets/svg/files/litecoin-icon.svg:
delete this file

timohanke commented on frontend/assets/svg/files/neo-icon.svg:
delete this file

timohanke commented on frontend/const.ts:
```diff
@@ -0,0 +1,157 @@
+// svg
+import IcIcon from "@/assets/img/icp.png";
+import EthereumIcon from "@/assets/svg/files/ethereum-icon.svg";
+import NeoIcon from "@/assets/svg/files/neo-icon.svg";
+import DecredIcon from "@/assets/svg/files/decred-icon.svg";
+import BinanceIcon from "@/assets/svg/files/binance-icon.svg";
+import LitecoinIcon from "@/assets/svg/files/litecoin-icon.svg";
```
Remove these lines for irrelevant altcoins

timohanke commented on frontend/const.ts:
```diff
@@ -0,0 +1,157 @@
+//
+export const ICRC1systemAssets: Array<Token> = [
+  {
+    id_number: 9999,
```
Is this ok to have multiple with id_number 9999? Can it cause any problems?

Jair-MoralesN replied:
Just a initial number, then when a token is added a new id will be asigned. However I'll change a different id for each.

timohanke commented on frontend/const.ts:
```diff
@@ -0,0 +1,157 @@
+  BTC: BitcoinIcon,
+  ICP: IcIcon,
+  ETH: EthereumIcon,
+  NEO: NeoIcon,
+  DCR: DecredIcon,
+  BNB: BinanceIcon,
+  LTC: LitecoinIcon,
```
remove these lines

timohanke commented on frontend/const.ts:
```diff
@@ -0,0 +1,157 @@
+export const EXTsystemAssets: Array<Token> = [];
+export const systemAssets: { [key: string]: Array<Token> } = {
+  "ICRC-1": ICRC1systemAssets,
+  DIP20: DIP20systemAssets,
+  EXT: EXTsystemAssets,
```
Do we use dip20 and ext for anything? Do we need these lines?

timohanke reviewed (commented):
Left some comments and questions

timohanke commented on README.md:
```diff
@@ -1 +1,107 @@
+
+## License
+
+This project is licensed under the GPL3 License - see the [COPYING](COPYING) file for details.
```
LICENSE file, not COPYING

Commit:
feat: :sparkles: Wallet functionality for ICRC 1 tokens

Wallet to manage their digital assets and perform transactions with major ICRC1 cryptocurrencies

User-friendly interface for ICRC 1 tokens
